### ISSUE TITLE
The game crashed whilst initializing game Error: java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'wurst'! Exit Code: 255

### ISSUE THREAD

**Issue by koweechi** [Wurst-Imperium/Wurst7#235]
Before posting, please make sure that this hasn't already been reported by someone else.

Hi, me and some friends decided to do an anarchy server so I tried installing Wurst, but when I open the game the Minecraft launcher tells me the message above.

Here's how I went on about it
1. I downloaded the Wurst mod for Fabric in Minecraft 1.16.4 as well as the Fabric.api. 
2. I made sure each and every file was up to date and that i was using the correct one.
3. I opened the Minecraft Launcher and launched Minecraft and after 30 seconds the message above appeared.
 
Sadly, I didn't take any screenshots

---- Minecraft Crash Report ----
// Sorry :(

Time: 11/26/20 4:40 PM
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'wurst'!
at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53)
at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36)
at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:32)
at net.minecraft.client.MinecraftClient.(MinecraftClient:437)
at net.minecraft.client.main.Main.main(Main:177)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:224)
at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:141)
at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27)
Caused by: java.lang.RuntimeException: Couldn't create '.Wurst encryption' folder.
at net.wurstclient.WurstClient.createEncryptionFolder(WurstClient.java:202)
at net.wurstclient.WurstClient.initialize(WurstClient.java:148)
at net.wurstclient.WurstInitializer.onInitialize(WurstInitializer.java:27)
at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50)
... 11 more
Caused by: java.nio.file.AccessDeniedException: /home/koichi/.Wurst encryption
at sun.nio.fs.UnixException.translateToIOException(UnixException.java:84)
at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
at sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:384)
at java.nio.file.Files.createDirectory(Files.java:674)
at java.nio.file.Files.createAndCheckIsDirectory(Files.java:781)
at java.nio.file.Files.createDirectories(Files.java:767)
at net.wurstclient.WurstClient.createEncryptionFolder(WurstClient.java:185)
... 14 more
A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Thread: Render thread
Stacktrace:
at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53)
at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36)
at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:32)
at net.minecraft.class_310.(class_310.java:437)

-- Initialization --
Details:
Stacktrace:
at net.minecraft.client.main.Main.main(Main.java:177)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:224)
at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:141)
at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27)

-- System Details --
Details:
Minecraft Version: 1.16.4
Minecraft Version ID: 1.16.4
Operating System: Linux (amd64) version 5.4.0-53-generic
Java Version: 1.8.0_275, Private Build
Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Private Build
Memory: 1073447944 bytes (1023 MB) / 2080374784 bytes (1984 MB) up to 2147483648 bytes (2048 MB)
CPUs: 8
JVM Flags: 8 total; -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
Fabric Mods:
fabric: Fabric API 0.25.4+1.16
fabric-api-base: Fabric API Base 0.2.0+9f0fb4b83a
fabric-biome-api-v1: Fabric Biome API (v1) 3.0.0+8367596d3a
fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.4+c6a8ea893a
fabric-command-api-v1: Fabric Command API (v1) 1.0.9+a4467d2a3a
fabric-commands-v0: Fabric Commands (v0) 0.2.0+52d308363a
fabric-containers-v0: Fabric Containers (v0) 0.1.8+045df74f3a
fabric-content-registries-v0: Fabric Content Registries (v0) 0.1.9+059ea8663a
fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.2+b7f9825d3a
fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.1+f8ac1db23a
fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.0+16acbe5b3a
fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.3+a4467d2a3a
fabric-item-api-v1: Fabric Item API (v1) 1.2.0+f8ac1db23a
fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.1+a4467d2a3a
fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.1+730711c63a
fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.0+3fa9f7c53a
fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.0+74cc3b203a
fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.1+432ea1883a
fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.2+b764ce993a
fabric-models-v0: Fabric Models (v0) 0.1.1+f8ac1db23a
fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.5+b50ffc7b3a
fabric-networking-v0: Fabric Networking (v0) 0.1.10+e00ecb5f3a
fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.9.2+a89534ab3a
fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.1+a4467d2a3a
fabric-particles-v1: fabric-particles-v1 0.2.2+fb0d9b0e3a
fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.6.1+a4467d2a3a
fabric-renderer-api-v1: Fabric Renderer API (v1) 0.3.3+a4467d2a3a
fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.3+a4467d2a3a
fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.2.0+f8ac1db23a
fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.4+f8ac1db23a
fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.12+f8ac1db23a
fabric-rendering-v0: Fabric Rendering (v0) 1.1.1+f8ac1db23a
fabric-rendering-v1: Fabric Rendering (v1) 1.3.1+f8ac1db23a
fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.3.1+facf3bbf3a
fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.0+872498413a
fabric-structure-api-v1: Fabric Structure API (v1) 1.1.0+f8ac1db23a
fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.0.3+ac8e8c593a
fabric-textures-v0: Fabric Textures (v0) 1.0.5+a4467d2a3a
fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.5+c93221003a
fabricloader: Fabric Loader 0.10.8
io_sentry_sentry: sentry 3.1.0
minecraft: Minecraft 1.16.4
wurst: Wurst Client v7.8-MC1.16.4
Launched Version: fabric-loader-0.10.8-1.16.4
Backend library: LWJGL version 3.2.2 build 10
Backend API: NO CONTEXT
GL Caps:
Using VBOs: Yes
Is Modded: Definitely; Client brand changed to 'fabric'
Type: Client (map_client.txt)
CPU: 

**Please note:** If the game did not crash, please hold down F3+C for 10 seconds to generate a crash report. Even when a bug doesn't cause the game to crash, this file still contains useful information that can help us to find and fix the problem. Because of this, please always include a crash report.

**Comment by koweechi** [comment]
Also if it helps, my OS right now is Ubuntu 20.04

**Comment by Alexander01998** [comment]
> Caused by: java.nio.file.AccessDeniedException: /home/koichi/.Wurst encryption

This seems to be the problem. Wurst is trying to create that folder, but access is denied.

I don't think it's an OS-specific problem, since I've seen the same thing happen on a Windows machine. This only happens to a handful of people for some strange reason.

Try creating the folder manually, that fixed it last time. If it still doesn't work after that, try `chmod`-ing the ".Wurst encryption" folder and its contents to make sure Wurst has access to it.

**Comment by koweechi** [comment]
> > Caused by: java.nio.file.AccessDeniedException: /home/koichi/.Wurst encryption
> 
> This seems to be the problem. Wurst is trying to create that folder, but access is denied.
> 
> I don't think it's an OS-specific problem, since I've seen the same thing happen on a Windows machine. This only happens to a handful of people for some strange reason.
> 
> Try creating the folder manually, that fixed it last time. If it still doesn't work after that, try `chmod`-ing the ".Wurst encryption" folder and its contents to make sure Wurst has access to it.

It didn't seem to work, I did the following commands in my console:
     cd snap/ cd mc-installer/ cd .minecraft/ chmod ugo+rwx .Wurst encryption

Did i do something wrong? I've never used the chmod command so i don't know much about it

**Comment by Alexander01998** [comment]
Try this:

```
sudo mkdir "/home/koichi/.Wurst encryption"
sudo chmod -R 777 "/home/koichi/.Wurst encryption"
```

**Comment by Alexander01998** [comment]
If that works, you should probably lower the permissions afterwards so that not everyone can access the folder.

Something like this:
```
sudo chown -R koichi:koichi "/home/koichi/.Wurst encryption"
sudo chmod -R 770 "/home/koichi/.Wurst encryption"
```

**Comment by koweechi** [comment]
Hmmm, tried that (filling in the rest of the directories leading up to .Wurst encryption) and it didn't work at all, guess I don't have any luck at all when it comes to computers

**Comment by Alexander01998** [comment]
> filling in the rest of the directories leading up to .Wurst encryption

I already gave you the full path. There was nothing to fill in.

**Comment by koweechi** [comment]
Ooohh, I thought the .Wurst encryption folder was in .minecraft, anyways I deleted the folder I created in .minecraft leaving only the .Wurst encryption folder in /home/koichi/.
When I tried opening the game it greeted me with a crash and the crash report was similar as the first one but instead of 
Caused by: java.nio.file.AccessDeniedException: /home/koichi/.Wurst encryption 
it had 
Caused by: java.nio.file.AccessDeniedException: /home/koichi/.Wurst encryption/READ ME I AM VERY IMPORTANT.txt
I'll try to do what I did before using chmod but I don't know if it'll work

**Comment by koweechi** [comment]
The terminal says:
   koichi@PCmatias:~$ sudo ls -la "/home/koichi/.Wurst encryption"
   [sudo] password for koichi: 
   total 12
   drwxrwx---  3 koichi koichi 4096 nov 30 10:57  .
   drwxr-xr-x 31 koichi koichi 4096 nov 30 12:42  ..
   drwxrwx---  2 koichi koichi 4096 nov 30 10:57 'READ ME I AM VERY IMPORTANT.txt'
   koichi@PCmatias:~$ sudo ls -la "/home/koichi/.minecraft/wurst"
   ls: cannot access '/home/koichi/.minecraft/wurst': No such file or directory
That is very wierd because there is clearly a wurst folder in .minecraft
![Screenshot from 2020-12-03 08-15-36](https://user-images.githubusercontent.com/69765892/101033275-c2106c00-353f-11eb-9572-56fb4d4671bc.png)

**Comment by koweechi** [comment]
I don't know why the text I copied from the terminal is in strikethru but I think it's readable.

**Comment by Alexander01998** [comment]
> I don't know why the text I copied from the terminal is in strikethru but I think it's readable.

I have fixed that for you.

>   drwxrwx---  3 koichi koichi 4096 nov 30 10:57  .
>   drwxr-xr-x 31 koichi koichi 4096 nov 30 12:42  ..
>   drwxrwx---  2 koichi koichi 4096 nov 30 10:57 'READ ME I AM VERY IMPORTANT.txt'

Those permissions look correct. The user `koichi` should have full access to those files. Maybe Wurst really is running as a different user?

> That is very wierd because there is clearly a wurst folder in .minecraft
> [screenshot]

If I'm interpreting that screenshot correctly, it looks like that folder is located at either
`/home/koichi/snap/mc-installer/518/.minecraft` or `/snap/mc-installer/518/.minecraft`.

You should be able to right click an empty spot in the file explorer window and then select something like "Open in Terminal". That would save us from having to guess the path.

![image](https://user-images.githubusercontent.com/10100202/101221636-5b4a9a00-3688-11eb-9b9a-a01809f7b5a4.png)
(This is from an old Ubuntu 16.04 VM I had lying around. Hopefully still works like that in 20.04.)

Then you can just do `sudo ls -la wurst` to see which user owns the wurst folder.

**Comment by koweechi** [comment]
The folder Its located in is ~/snap/mc-installer/518/.minecraft
But I have no idea about whats happening to wurst
![Screenshot from 2020-12-05 09-12-14](https://user-images.githubusercontent.com/69765892/101246648-4dbe0000-36da-11eb-893b-38844cf31acd.png)

**Comment by Alexander01998** [comment]
That's so weird. I also have no idea why this isn't working. Everything looks correct.

**Comment by Nikcss** [comment]
---- Minecraft Crash Report ----
// Uh... Did I do that?

Time: 14/05/21 15:45
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'blade'!
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:32)
	at net.minecraft.class_310.<init>(class_310.java:437)
	at net.minecraft.client.main.Main.main(Main.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
Caused by: java.lang.NoClassDefFoundError: kotlin/jvm/internal/Intrinsics
	at vini2003.xyz.blade.Blade.<clinit>(Blade.java:19)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at net.fabricmc.loader.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:45)
	at net.fabricmc.loader.EntrypointStorage$NewEntry.create(EntrypointStorage.java:112)
	at net.fabricmc.loader.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:99)
	at net.fabricmc.loader.EntrypointStorage.lambda$getEntrypointContainers$1(EntrypointStorage.java:184)
	at net.fabricmc.loader.EntrypointStorage$$Lambda$2243/709685969.get(Unknown Source)
	at net.fabricmc.loader.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:37)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50)
	... 11 more
Caused by: java.lang.ClassNotFoundException: kotlin.jvm.internal.Intrinsics
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:168)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 21 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:32)
	at net.minecraft.class_310.<init>(class_310.java:437)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.main.Main.main(Main.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)

-- System Details --
Details:
	Minecraft Version: 1.16.5
	Minecraft Version ID: 1.16.5
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 940655896 bytes (897 MB) / 1845493760 bytes (1760 MB) up to 2147483648 bytes (2048 MB)
	CPUs: 6
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		added_origins_mod: Added Origins V1.0.0
		appleskin: AppleSkin 1.0.11
		autoconfig1u: Auto Config v1 Updated 3.3.1
		blade: Blade 2.0.1+1.16.5
		bodyshuffle: Body Shuffle 1.0.4+1.16.5
		byg: BYG 1.1.7
		cardinal-components: Cardinal Components API 2.7.11
		cardinal-components-base: Cardinal Components API (base) 2.7.11
		cardinal-components-block: Cardinal Components API (blocks) 2.7.11
		cardinal-components-chunk: Cardinal Components API (chunks) 2.7.11
		cardinal-components-entity: Cardinal Components API (entities) 2.7.11
		cardinal-components-item: Cardinal Components API (items) 2.7.11
		cardinal-components-level: Cardinal Components API (world saves) 2.7.11
		cardinal-components-scoreboard: Cardinal Components API (scoreboard) 2.7.11
		cardinal-components-util: Cardinal Components API (utilities) 2.7.11
		cardinal-components-world: Cardinal Components API (worlds) 2.7.11
		chestcavity: Chest Cavity 2.10.1
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-config2: Cloth Config v4 4.11.14
		cursedorigins: CursedOrigins 1.1.4
		daemonic: Daemonic 1.1.1
		fabric: Fabric API 0.32.5+1.16
		fabric-api-base: Fabric API Base 0.2.1+9354966b7d
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.0.0+dc716ea17d
		fabric-biome-api-v1: Fabric Biome API (v1) 3.1.1+ca58154a7d
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.5+ca58154a7d
		fabric-command-api-v1: Fabric Command API (v1) 1.1.1+351679a77d
		fabric-commands-v0: Fabric Commands (v0) 0.2.2+ca58154a7d
		fabric-containers-v0: Fabric Containers (v0) 0.1.11+9354966b7d
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.1+ca58154a7d
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.3+ca58154a7d
		fabric-dimensions-v1: fabric-dimensions-v1 2.0.6+9354966b7d
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.0.3+ca58154a7d
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.2+ca58154a7d
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+ca58154a7d
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.6+ca58154a7d
		fabric-item-api-v1: Fabric Item API (v1) 1.2.1+ca58154a7d
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.5+351679a77d
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.4+9354966b7d
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.1+ca58154a7d
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.1+ca58154a7d
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.2+ca58154a7d
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+ca58154a7d
		fabric-models-v0: Fabric Models (v0) 0.2.1+ca58154a7d
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.1+ca58154a7d
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.8+ca58154a7d
		fabric-networking-v0: Fabric Networking (v0) 0.3.2+ca58154a7d
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.9.4+9354966b7d
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.2+ca58154a7d
		fabric-particles-v1: Fabric Particles (v1) 0.2.4+ca58154a7d
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.4+ca58154a7d
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.1+ca58154a7d
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.4+ca58154a7d
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.2.1+ca58154a7d
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.5+ca58154a7d
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.13+ca58154a7d
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.2+ca58154a7d
		fabric-rendering-v1: Fabric Rendering (v1) 1.5.1+ca58154a7d
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.2+ca58154a7d
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.0+c045166c7d
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.5+9354966b7d
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.4+ca58154a7d
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.1+ca58154a7d
		fabric-textures-v0: Fabric Textures (v0) 1.0.6+ca58154a7d
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.6+ca58154a7d
		fabricloader: Fabric Loader 0.11.3
		fallflyinglib: FallFlyingLib 1.1.0
		inventoryhud: Inventory HUD + 3.2.2
		java: Java HotSpot(TM) 64-Bit Server VM 8
		meumo: Mega Extra Ultimate Mob Origins 1.0.1
		minecraft: Minecraft 1.16.5
		moborigins: Mob Origins 1.4.0
		modify-drops-api: Modify Drops API 0.3.1
		modmenu: Mod Menu 1.16.8
		origins: Origins 0.7.0
		origins-classes: Origins: Classes 1.1.1
		originsplus: Origins Plus 0.0.2
		pehkui: Pehkui 2.0.0+21w14a
		reach-entity-attributes: Reach Entity Attribute 1.0.1
		satin: Satin 1.5.1
		slimeorigin: Slimeorigin 3.0.0
		sodium: Sodium 0.1.0
		tk_valoeghese_zoesteriaconfig: ZoesteriaConfig 1.3.2
		toomanyorigins: TooManyOrigins 0.1.2
		torohealth: ToroHealth Damage Indicators 1.16.4-fabric-4
		voidwalker: Voidwalker 1.16.5-8
		worldedit: WorldEdit 7.2.4+5723-e3bfc64
		xaerominimap: Xaero's Minimap 21.8.0
	Launched Version: fabric-loader-0.11.3-1.16.5
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: NO CONTEXT
	GL Caps: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: <unknown>

**Comment by Alexander01998** [comment]
This is mostly fixed as of [Wurst 7.22](https://www.wurstclient.net/updates/wurst-7-22/).

Having this weird permission issue will now only stop AltManager from saving the alt list. Everything else remains usable and the game no longer crashes.
